src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -124,12 +120,10 @@ const myAppConfig = {
 Amplify.configure(myAppConfig);
 ```
 
-[question]: # (Do we want to address non-AppSync on this job or link to it? Or does this become an Expander section?)
+<Accordion title='Using a non-AppSync GraphQL server' headingLevel='3' eyebrow='Learn more'>
```

Comment:
I missed asking this Q again in our Slack conversation but I made the choice to put this next section in an expander vs on the page. Does this work or do we want to take this option out of the expander and list AppSync as Option 1 and non-AppSync as Option 2?

Reply:
Okay as expander. It's not a recommended path for most customers.